docker/django/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.4
+
+ENV PYTHONUNBUFFERED 0
+
+RUN apt-get update \
+    && apt-get install -y gdal-bin python-gdal python3-gdal
+
+# Let's install git packages from requirements.txt here
+RUN mkdir -p /src
```

Comment:
Instead of this, you can shorten the dockerfile a bit and use /usr/src

Reply:
oh, yeah, good idea! will use that 🙂